spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -115,21 +115,21 @@ module.exports = function (provider, setup) {
       var paused = false;
       var messageCount = 0;
       dispatch.on('onMessage', function (msg) {
-        if (!('data' in msg)) {
+        if (msg && !(msg.hasOwnProperty('data'))) {
```

Comment:
Why would `msg` be null?  Is Firefox sending us null `msg`?  That seems like a bug.